.gitignore

```diff
@@ -10,3 +10,5 @@ local.properties
 build
 **/*.iml
 target
+dependency-reduced-pom.xml
```

Comment:
We do use this pom, but it's generated by the plugin and doesn't need to be committed.